features/mbedtls/targets/TARGET_CRYPTOCELL310/Readme.md

```diff
@@ -0,0 +1,22 @@
+# Guidelines for porting Cryptocell to Mbed OS
+
+Cryptocell (CC) 310 is a hardware accelerator that can be ported to several boards, assuming these boards have CC 310 embedded in their hardware.
+
+The CC 310 driver consists of three libraries:
+
+* A common core library(`libcc_core.a`).
+* A platform-specific TRNG library, containing TRNG-related information, for sampling sufficient entropy on the specific platform(`libcc_trng.a`).
+* A platform-specific library containig extra information, such as the CC register's base addresses on the specific board. (`libcc_ext.a`)
+
+To port your CC 310 driver to Mbed OS on your specific target, do the following:
+
+1. In `targets.json` add the following  to your target:
+ 	* `MBEDTLS_CONFIG_HW_SUPPORT` to `macros_add` key.
+ 	* `CRYPTOCELL` to `device_has_add` key.
+ 	* `CRYPTOCELL310` to `extra_labels_add` key.
+1. In `objects.h`, include `objects_cryptocell.h`. (You can condition it with `#if DEVICE_CRYPTOCELL` in case you have another `trng` engine for a differnt board, and `objects.h` is common file for your boards, in this case your common `trng_api.c` file should be compiled only if `#if !defined(DEVICE_CRYPTOCELL)`).
```

Comment:
This line confuses me. Could you please rewrite the second and third sentences for clarity?